src/App.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from "react";
+import { Container, Footer } from "./styledComponents";
+import Weather from "./Weather";
+import Home from "./Home";
+import ErrorPage from "./Error";
+import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
+
+class App extends Component {
+  render() {
+    return (
+      <Router>
+        <Container>
+          <Switch>
+            <Route exact path="/" component={Home} />
+            <Route exact path="/:city/:state/:ip" component={Weather} />
```

Comment:
Can the `:/city/:state/` route be re-added? Maybe at some point we can add zipcode if we want to be more specific. I think that looks more polished than having the `ip` in the url.

Reply:
> Can the `:/city/:state/` route be re-added? Maybe at some point we can add zipcode if we want to be more specific. I think that looks more polished than having the `ip` in the url.

Sure, please see my previous comment on how I would have to approach this.